src/forcing/ForcingsEngineLumpedDataProvider.cpp

```diff
@@ -0,0 +1,134 @@
+#include "DataProvider.hpp"
+#include <chrono>
+#include <forcing/ForcingsEngineLumpedDataProvider.hpp>
+
+namespace data_access {
+
+using Provider     = ForcingsEngineLumpedDataProvider;
+using BaseProvider = Provider::base_type;
+
+std::size_t convert_divide_id_stoi(const std::string& divide_id)
+{
+    auto separator = divide_id.find('-');
+    const char* split = (
+        separator == std::string::npos
+        ? &divide_id[0]
+        : &divide_id[separator + 1]
+    );
+
+    return std::atol(split);
```

Comment:
I normally prefer strtol() to atol(). For most linux compiliers the actual conversion will be a call internal call to strtol so it is easier for strtol calls to be implemented.

Reply:
Oh this is a good point -- plus, thinking about it a it more, strtol() is safer. I'll make an issue as a reminder to change this. Thanks 😄